filebeat/harvester/log.go

```diff
@@ -63,8 +64,7 @@ func (h *Harvester) Harvest() {
 		default:
 		}
 
-		// Partial lines return error and are only read on completion
-		ts, text, bytesRead, jsonFields, err := readLine(r)
+		message, err := r.Next()
```

Comment:
normally new lines characters are added to byte count, so to have correct offset.